### ISSUE TITLE
when inserting with postgres, nothing is returned

### ISSUE THREAD

**Issue by verybigelephants** [catfan/Medoo#386]
when using psql as the engine, if you do 

```
$last_id=$database->insert('sometable', array(..))
```

`$last_id` contains nothing :(

http://medoo.in/api/insert says it should return the latest inserted id
i have created the table with the primary key like this 

```
CREATE TABLE memes (
    sometable_id serial primary key,
    ...
)
```

**Comment by dennis90** [comment]
I fixed that as you can see in PR: #431,
the problem source: http://php.net/manual/en/pdo.lastinsertid.php#75200

**Comment by catfan** [comment]
The return id is fixed now.
